master/docs/relnotes/index.rst

```diff
@@ -43,6 +43,8 @@ Fixes
 
 * sqlite access is serialized in order to improve stability (:bug:`3565`)
 
+* Docker latent worker has been fixed (:bug:`3571`)
```

Comment:
Well this feature never worked

Le mar. 28 juin 2016 11:51, Vladimir Rutsky <notifications@github.com> a
écrit :

> In master/docs/relnotes/index.rst
> <https://github.com/buildbot/buildbot/pull/2296#discussion_r68728334>:
>
> > @@ -43,6 +43,8 @@ Fixes
> >
> >  * sqlite access is serialized in order to improve stability (:bug:`3565`)
> >
> > +* Docker latent worker has been fixed (:bug:`3571`)
>
> @tardyp <https://github.com/tardyp> actually I thought that you will
> document that a feature was removed :)
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/buildbot/buildbot/pull/2296/files/d2a4711cdd6f783724e3c2d6ba41cf4dc85c93e1#r68728334>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAGtI5ToJVkA12BkBTNnFuzwQxGzRtq2ks5qQO6LgaJpZM4I_UAo>
> .
>